roles/openshift_node/files/clean-up-crio-pods.sh

```diff
@@ -18,7 +18,6 @@ for c in $(runc list -q); do
         done
 done
 mount | grep overlay | awk '{print $3}' | xargs umount | true
-umount -R /var/lib/containers/storage/overlay
-umount -R /var/lib/containers/storage
+mount | grep '/var/lib/containers/storage/' | awk '{print $3}' | xargs umount | true
```

Comment:
I still don't understand how this change relates to a problem/solution with the BZ.

BZ complains we're deleting stuff... this patch is still deleting stuff.  Please update commit message with problem/solution statement because I don't understand what those are.

Reply:
@michaelgugino Done.